src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/AuthorizationFunctionalTest.java

```diff
@@ -58,14 +65,12 @@
     protected static String s2sToken;
 
     public static final String EMAIL = "EMAIL";
-
     public static final String CREDS = "CREDS";
-
-    public static final String BEARER = "BEARER";
-
     public static final String EMAIL_TEMPLATE = "CWR-func-test-user-%s@cwrfunctestuser.com";
```

Comment:
do we need to change the domain to @justice.gov.uk?

Reply:
good point, changed